packages/api-codec/src/Metadata.ts

```diff
@@ -285,9 +287,9 @@ export default class RuntimeMetadata extends Struct {
 
   // We receive this as an Array<number> in the JSON output from the Node. Convert
   // to u8a and use the fromU8a to do the actual parsing
-  fromJSON (input: Array<number>): RuntimeMetadata {
+  fromJSON (input: Uint8Array | string | Array<number>): RuntimeMetadata {
```

Comment:
The actual JSON has changed, it now comes in not as an `Array<number>`, but rather as a hex string. Cater for all.